### PR TITLE
Use new @gravitee/ui-particles-angular lib 

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**

https://github.com/gravitee-io/issues/issues/6444

**Description**

Configure console ui with ivy 

Use new lib @gravitee/ui-particles-angular and use it 

**Additional context**

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-ttcvgwfijk.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/ui-particles/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: gravitee-apim-console-webui/.npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+@gravitee-io:registry=https://npm.pkg.github.com
```
Comment (gaetanmaisse) [inline gravitee-apim-console-webui/.npmrc]: not needed anymore

File: gravitee-apim-console-webui/src/index.scss [hunk 2]
```diff
@@ -127,10 +126,14 @@ body {
 // Wrap mat with `.mat` class without increasing the css specificity
 // Allows you to disable it on some routes
 :where(.mat) {
-  @include mat.core($gio-typography);
-  @include mat.all-component-themes($gio-theme);
+  // Material theme config from Gravitee ui particles
+  @include gio.mat-theme();
+
+  // Scss component from Gravitee ui particles
+  @include gio.all-component();
```
Comment (gaetanmaisse) [inline gravitee-apim-console-webui/src/index.scss]: To update after https://github.com/gravitee-io/gravitee-ui-particles/pull/8/commits/1aacb8f4fd3802e382e9d90651906cdcb3355973 is merged

**Review by gaetanmaisse — APPROVED** [review]

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-ietycxlydp.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/ui-particles/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-xakrduilje.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/ui-particles/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Pull request merged**